pkgs/applications/graphics/animbar/default.nix

```diff
@@ -1,39 +0,0 @@
-{ stdenv, lib, fetchurl, cmake, qt4, file }:
```

Comment:
We have a [Qt 5 patch](https://github.com/NixOS/nixpkgs/pull/44608#issuecomment-421875718) but upstream never responded so dropping it is probably fine.

Reply:
whichever option you prefer is fine by me